crates/red_knot_python_semantic/resources/mdtest/call/builtins.md

```diff
@@ -0,0 +1,14 @@
+# Calling builtins
+
+## `bool` with incorrect arguments
+
+```py
+class NotBool:
+    __bool__ = None
+
+# TODO: We should emit an `invalid-argument` error here for `2` because `bool` only takes one argument.
```

Comment:
this is more something I noticed and considered worth adding a test. It isn't itself related to making `Type::bool` faliable